test/31.js

```diff
@@ -18,4 +19,20 @@ describe('longest string in an array', () => {
   it('the longest string in [just] is just', () => {
     expect(solution(['just'])).eql('just');
   })
+  it.only('the longest string in [hi,hello,hola] is hello', () => {
+    expect(solution1(['hi','hello','hola'])).eql('hello');
```

Comment:
Instead of creating 5 literally duplicate test cases, could you just take advantage of the test cases already written?